### PR TITLE
MG-3014 Update package version in data-detective-airflow

### PR DESCRIPTION
**Pull request by ikanashov**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [x] Refactoring (no functional changes, no api changes)
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #45 

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

## Other information

Need to update data-detective-etl after merge to master

### CODE REVIEW

File: data-detective-airflow/pyproject.toml [hunk 1]
```diff
@@ -27,9 +27,9 @@ apache-airflow-providers-celery = "*"
 apache-airflow-providers-postgres = "*"
 apache-airflow-providers-redis = "*"
 apache-airflow-providers-ssh = "*"
-pandas = "^1.1"
-petl = "^1.7"
-psycopg2-binary = "^2.8"
+pandas = "~1.3.5"
```
Comment (mebelousov) [inline data-detective-airflow/pyproject.toml:30]: Обязательно апать версии? Библиотека должна быть гибкой, подходить к разным версиям.
https://python-poetry.org/docs/dependency-specification/

File: data-detective-airflow/data_detective_airflow/utils/notnull.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-from collections import Iterable
+from collections.abc import Iterable
```
Comment (mebelousov) [inline data-detective-airflow/data_detective_airflow/utils/notnull.py:1]: Это единственное ограничение для запуска на python 3.10? Если теперь можно запустить на 3.10, то просьба добавить запуск тестов в дополнении к 3.9.

File: data-detective-airflow/pyproject.toml (hunk 1 above)
Comment (ikanashov) [reply data-detective-airflow/pyproject.toml]: done

File: data-detective-airflow/data_detective_airflow/utils/notnull.py (hunk 2 above)
Comment (ikanashov) [reply data-detective-airflow/data_detective_airflow/utils/notnull.py:1]: done

**Review by mebelousov — APPROVED** [review]
Спасибо!

**Pull request merged**